test/unittests/oom_test.cpp

```diff
@@ -25,7 +25,7 @@
 
 namespace
 {
-constexpr uint32_t OSMemoryLimit = 1024 * 1024;
+constexpr uint32_t OSMemoryLimit = 2 * 1024 * 1024;
```

Comment:
If this is enough only for running single test, make it bigger, e.g. 10M.

Reply:
No, for me it passes everything with this.

Reply:
Single test was passing with 1Mb.